docs/csharp/language-reference/compiler-messages/toc.yml

```diff
@@ -1561,6 +1561,8 @@ items:
     href: cs8410.md
   - name: CS8411
     href: cs8411.md
+- name: Warning waves
```

Comment:
Great catch @gewarren I can't believe I missed adding this.

This new article should replace the node "Level 5 warning messages", starting at line 1882. I'll add a second comment there.

I can open a separate PR for this if you want.